client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -19,26 +19,28 @@ import PromotionAppliesToField from './fields/promotion-applies-to-field';
 /**
  * "Coupon code" field reused for all coupon promotion types.
```

Comment:
Hi! I've found a possible matching string that has already been translated **51** times:
`translate( 'Enter Coupon Code' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/promotions-validation)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).